mlflow/pipelines/utils/step.py

```diff
@@ -66,6 +68,16 @@ def display_html(html_data: str = None, html_file_path: str = None) -> None:
         html_file_path = html_file_path if html_data is None else None
 
         if is_in_databricks_runtime():
+            from packaging import version
+
+            dbr_version_image_key = get_databricks_runtime()
+            dbr_version = dbr_version_image_key.split("-")[0]
+            if version.parse(dbr_version) < version.parse("11.x"):
```

Comment:
This block may not work properly because `packaging` only works for python package versions.

Reply:
```python
>>> from packaging import version
>>> version.parse("10.0") < version.parse("11.x")
False
```

Reply:
Does `dbr_version` always looks like `<num>-x`?

Reply:
Great catch! Updated to `            if LooseVersion(dbr_version).version[0] < 11:`, which seems to work

Reply:
```python
if int(dbr_version_image_key.split(".")[-1]) < 11:
```

also works (as long as `dbr_version_image_key` matches `^\d+\.`)